### PR TITLE
[TRAFODION-2889] Add Examples of LOAD Statement for *Trafodion SQL Reference Manual* 

### PR DESCRIPTION
**Pull request by liuyu000**

### CODE REVIEW

File: docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc [hunk 1]
```diff
@@ -650,6 +650,111 @@ Tables must be created/dropped/altered through Hive itself.
 [[load_examples]]
 === Examples of LOAD
 
+* This example demonstrates the LOAD statement continues loading data from source_table into target_table after ignorable data conversion error.
```
Comment (DaveBirdsall) [inline docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc]: Grammar is not quite right... adding the word "how" just before "the LOAD statement" will fix it. But this doesn't quite match the content below... the content shows what happens when we don't supply CONTINUE ON ERROR, then it shows a contrasting example with CONTINUE ON ERROR. Possible wordsmith: "The examples below demonstrate how the LOAD statement behaves without and with CONTINUE ON ERROR, when ignorable data conversion errors occur."

File: docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc [hunk 2]
```diff
@@ -650,6 +650,111 @@ Tables must be created/dropped/altered through Hive itself.
...
+  , B                                CHAR(3) CHARACTER SET ISO88591 COLLATE
+      DEFAULT DEFAULT NULL NOT SERIALIZED
+  )
+ ATTRIBUTES ALIGNED FORMAT 
+;
+--- SQL operation complete.
+```
+
++
+Load data from source_table into target_table when `CONTINUE ON ERROR` is not specified, the operation fails with data conversion error.
```
Comment (DaveBirdsall) [inline docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc]: Grammar. Suggest replacing "Load data..." with "When loading data...". Also "a data conversion error" is slightly better.

File: docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc [hunk 3]
```diff
@@ -650,6 +650,111 @@ Tables must be created/dropped/altered through Hive itself.
...
+Task:  LOAD            Status: Started    Object: TRAFODION.SEABASE.TARGET_TABLE                                                          
+Task:  CLEANUP         Status: Started    Time: 2018-01-03 16:15:53.222441                                                      
+Task:  CLEANUP         Status: Ended      Time: 2018-01-03 16:15:53.250826
+Task:  CLEANUP         Status: Ended      Elapsed Time:    00:00:00.028
+Task:  LOADING DATA    Status: Started    Time: 2018-01-03 16:15:53.250909                                                      
+*** ERROR[8402] A string overflow occurred during the evaluation of a character expression. Conversion of Source Type:CHAR(REC_BYTE_F_ASCII,4 BYTES,ISO88591) Source Value:aaaa to Target Type:CHAR(REC_BYTE_F_ASCII,3 BYTES,ISO88591). [2018-01-03 16:15:54]
+```
+
++
+Load data from source_table into target_table when `CONTINUE ON ERROR` is specified, the operation succeeds after ignorable data conversion error.
```
Comment (DaveBirdsall) [inline docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc]: "When loading data..." is better. Also "ignorable data conversion errors" is better.

File: docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc [hunk 4]
```diff
@@ -650,6 +650,111 @@ Tables must be created/dropped/altered through Hive itself.
 [[load_examples]]
 === Examples of LOAD
 
+* This example demonstrates the LOAD statement continues loading data from source_table into target_table after ignorable data conversion error.
+
++ 
+Suppose that we have two tables (source_table and target_table) like this:
```
Comment (traflm) [inline docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc]: remove the 'like this'?
Or 'like these'?
I am not sure :-)

File: docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc (hunk 1 above)
Comment (liuyu000) [reply docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc]: Thanks Dave, your comments have been incorporated :)

File: docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc (hunk 2 above)
Comment (liuyu000) [reply docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc]: Thanks Dave, your comments have been incorporated :)

File: docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc (hunk 3 above)
Comment (liuyu000) [reply docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc]: Thanks Dave, your comments have been incorporated :)

File: docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc (hunk 4 above)
Comment (liuyu000) [reply docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc]: Thanks Ming :)
"this" represents the following situation (whole, singular).

Two similar examples from other manuals are attched for your reference:
1. The example in _Oracle SQL Reference Manual_
like this 1

2. The example in _MySQL SQL Reference Manual_
like this 2

Comment (traflm) [reply docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc]: you are right ^_^
I learned something new.

**Pull request merged**